netdisco/discoverables/yeelight.py

```diff
@@ -0,0 +1,18 @@
+"""Discover Yeelight bulbs, based on Kodi discoverable."""
+from . import MDNSDiscoverable
+
+
+# pylint: disable=too-few-public-methods
+class Discoverable(MDNSDiscoverable):
+    """Add support for discovering Yeelight."""
+
+    def __init__(self, nd):
+        """Initialize the Yeelight discovery."""
+        super(Discoverable, self).__init__(nd, '_miio._udp.local.')
+
+    def info_from_entry(self, entry):
+        """Return most important info from mDNS entries."""
+        return (self.ip_from_host(entry.server), entry.port)
```

Comment:
Great, sounds good, I'll change that.

Did you have in mind to have some generic keys shared among devices, or shall it be freeform and for plugins to deal with?

{"host": <ip>,
 "port": <port>,
 "hostname": "yeelink-light-color1_miioXXXX"|"yeelink-light-mono1_miioXXXX"
}

was what I was thinking for this case (if not even adding bulb_type=['rgb', 'white']) or some such about supported features.